benchmarks/src/test/java/org/apache/druid/benchmark/lookup/LookupBenchmarkUtil.java

```diff
@@ -77,7 +77,7 @@ public LookupExtractor build(Iterable<Pair<String, String>> keyValuePairs)
         return new MapLookupExtractor(map, false);
       }
     },
-    REVERSIBLE {
+    IMMUTABLE {
```

Comment:
Why is it called immutable?

Reply:
It's the immutable lookup map added in #15675